src/core/connection.c

```diff
@@ -1170,7 +1170,7 @@ QuicConnTimerSet(
     _In_ uint64_t Delay
```

Comment:
should this parameter be named `DelayUs` to indicate the expected resolution?

Reply:
It could be, but I think I'd have to update CLOG too since it's a logged parameter, and I don't want to have to do that. :D